independent-projects/bootstrap/core/src/test/java/io/quarkus/bootstrap/resolver/ResolverSetupCleanup.java

```diff
@@ -24,7 +25,7 @@
     public void setup() throws Exception {
         workDir = initWorkDir();
         repoHome = IoUtils.mkdirs(workDir.resolve("repo"));
-        resolver = initResolver();
+        resolver = initResolver(null);
```

Comment:
We really want to pass null here?

Reply:
Yes, it means we are not setting up a workspace reader.